kernel/inductive.mli

```diff
@@ -74,8 +72,13 @@ val arities_of_constructors : pinductive -> mind_specif -> types array
 (** Return constructor types in user form *)
 val type_of_constructors : pinductive -> mind_specif -> types array
 
-(** Transforms inductive specification into types (in nf) *)
-val arities_of_specif : MutInd.t puniverses -> mind_specif -> types array
+(** Turns a constructor type recursively referring to inductive types
+    into the same constructor type referring instead to a context made
+    from the abstract declaration of the inductive types (e.g. turns
+    [nat->nat] into [Rel 1 -> Rel 1]); takes as arguments the number
+    of inductive types in the block and the name of the block *)
+val abstract_constructor_type_relatively_to_inductive_types_context :
```

Comment:
That's a pretty long name ;)
I don't have a better suggestion so it will have to do

Reply:
I did not check, but it seems to be used only to port old code. If so, it should be probably deprecated no?

Reply:
One issue in porting plugins is that some (paramcoq, template_coq, ...) produces a `mind_entry_lc` from a `mind_user_lc`. It is not clear to me how easy it would be to impose that `mind_entry_lc` takes a reference to `Ind` rather than `Rel` in the types. So, a conversion function from `Ind` to `Rel` remains needed in this case.

Note that I used it also in the checker (for the same reason that the checker rebuilds a `mind_entry`.